pkgs/development/compilers/rust/rustc.nix

```diff
@@ -1,6 +1,6 @@
 { stdenv, removeReferencesTo, pkgsBuildBuild, pkgsBuildHost, pkgsBuildTarget
 , fetchurl, file, python2
-, llvm_7, darwin, git, cmake, rustPlatform
+, llvm_9, darwin, git, cmake, rustPlatform
```

Comment:
I think we can switch to `llvmPackages_latest.llvm` instead.

Reply:
Maybe it will be better to stick with version 9 and bump when rust upstream switches to newer version. I don't have strong opinion on this one.

Reply:
In that case please leave a note in rustc for maintainers to also check for the used llvm version when updating rustc.